.scripts/interactive_translate/main.py

```diff
@@ -0,0 +1,224 @@
+                translation = self.translate_text(prompt)
+                if translation:
+                    self.query_one("#translation-input").value = translation
+                    self.notify("Translation generated!")
+                else:
+                    self.notify("Failed to generate translation", severity="error")
+            except Exception as e:
+                self.notify(f"Error: {str(e)}", severity="error")
+        elif event.button.id == "save-btn":
+            self.po_file.save(str(self.po_file_path))
+            try:
+                subprocess.run(['powrap', str(self.po_file_path)], check=True)
+                self.notify("Changes saved and wrapped successfully!")
+            except subprocess.CalledProcessError:
+                self.notify("Save successful, but powrap failed!", severity="error")
+            except FileNotFoundError:
+                self.notify("Save successful, but powrap not found!", severity="warning")
+
+    def navigate_entry(self, direction: int) -> None:
+        new_index = self.current_entry_index + direction
+        if 0 <= new_index < len(self.po_file):
+            self.current_entry_index = new_index
+            self.show_current_entry()
+            self.query_one("#translation-input").value = ""
+
+    def action_previous(self) -> None:
+        self.navigate_entry(-1)
+
+    def action_next(self) -> None:
+        self.navigate_entry(1)
+
+    def action_accept(self) -> None:
+        translation = self.query_one("#translation-input").value
+        if translation:
+            self.po_file[self.current_entry_index].msgstr = translation
+            self.navigate_entry(1)
+
+    def action_save(self) -> None:
+        self.po_file.save(str(self.po_file_path))
+        try:
+            subprocess.run(['powrap', str(self.po_file_path)], check=True)
+            self.notify("Changes saved and wrapped successfully!")
+        except subprocess.CalledProcessError:
+            self.notify("Save successful, but powrap failed!", severity="error")
+        except FileNotFoundError:
+            self.notify("Save successful, but powrap not found!", severity="warning")
+            
+    def action_generate(self) -> None:
+        entry = self.po_file[self.current_entry_index]
+        try:
+            prompt = ("Translate the following Python documentation into Traditional Chinese"
+                f"for {self.po_file_path}:{entry.occurrences} with message {entry.msgid[1]}. Ensure "
+                "that the translation is accurate and uses appropriate technical terminology. The"
+                " output must be in Traditional Chinese. Pay careful attention to context, idiomatic"
+                " expressions, and any specialized vocabulary related to Python programming. Maintain "
+                "the structure and format of the original documentation as much as possible to ensure"
+                " clarity and usability for readers.")
```

Comment:
```suggestion
            prompt = ("Translate the following Python documentation into Traditional Chinese "
                f"for {self.po_file_path}:{entry.occurrences} with message {entry.msgid[1]}. Ensure "
                "that the translation is accurate and uses appropriate technical terminology. The "
                "output must be in Traditional Chinese. Pay careful attention to context, idiomatic "
                "expressions, and any specialized vocabulary related to Python programming. Maintain "
                "the structure and format of the original documentation as much as possible to ensure "
                "clarity and usability for readers.")
```
There's a missing space at the end of the first line that causes `Chinesefor` to be merged together.  While I was at it I made the spacing of the other lines consistent, by moving all the spaces at the end of the sentences.